src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -74,7 +74,8 @@ internal IVariableInitializerOperation CreateVariableDeclaratorInitializer(Bound
                     initializerIsImplicit = true;
                 }
 
-                return new CSharpLazyVariableInitializerOperation(this, boundLocalDeclaration.InitializerOpt, _semanticModel, initializerSyntax, type: null, constantValue: null, initializerIsImplicit);
+                IOperation value = Create(boundLocalDeclaration.InitializerOpt);
+                return new VariableInitializerOperation(locals: ImmutableArray<ILocalSymbol>.Empty, value, _semanticModel, initializerSyntax, initializerIsImplicit);
```

Comment:
Can you explain why we're passing an empty `IlocalSymbol` array here? That didn't match my expectations.

Reply:
These locals refer to `ISymbolInitializer.Locals`, which are locals that are scoped to just this initializer. Those don't exist for local variable initializers.